Cargo.toml

```diff
@@ -33,9 +33,10 @@ backtraces = ["cosmwasm-std/backtraces"]
 library = []
 
 [dependencies]
-cosmwasm-std = { version = "0.16.0" }
+cosmwasm-std = { version = "0.16.2" }
 cosmwasm-storage = { version = "0.16.0" }
 cw-storage-plus = "0.8.0"
+cw2 = { path = "../../packages/cw2", version = "0.8.1" }
```

Comment:
You need to remove `path` here, as that only works for packages inside the cw-plus monorepo.

Reply:
Ah, that's right. Changed.